distribution/std-lib/Standard/src/Base/Data/Array/Extensions.enso

```diff
@@ -1,8 +1,18 @@
 from Standard.Base import all
 
-## Transform the array into text for displaying as part of its default
-   visualization.
+## UNSTABLE
+   ADVANCED
+
+   Returns a Text used to display this value in the IDE.
+
+   The particular representation is left unspecified and subject to change in
+   the future. The current implementation uses JSON serialization as the
+   default.
+
+   > Example
+     Converting an array to its default visualization representation.
+         [1, 2, 3, 4].to_array.to_default_visualization_data
```

Comment:
This is ok, but just to ask - do we really need examples for these internal-facing functions? Shouldn't it actually be private since it is used just by the IDE?